src/components/Tabber/PreviewModeSelector/PreviewModeSelector.components.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import Icon from '~components/Icon/Icon';
+
+import scss from './PreviewModeSelector.module.scss';
+
+import { PreviewModeProps } from './PreviewModeSelector.types';
+
+export function PreviewMode(props: PreviewModeProps): React.JSX.Element {
+  return (
+    <div key={props.item.id}>
+      <input
+        className={scss.previewModeInput}
+        id={props.item.id}
+        name="device"
+        onChange={props.onSelect}
+        type="radio"
+        value={props.item.id}
+      />
+
+      <label
+        aria-label={props.item.name}
```

Comment:
eu tô vendo bem se esse aria-label é no input ou no label, pra não travar a revisão do pr, eu te respondo isso dps, mas se quiser dar uma pesquisada, é massa. mas to achando fortemente q esse aria-label vai pro input 🤔  vamos ver